docs/tutorial/02_deploying_magma_orchestrator.md

```diff
@@ -0,0 +1,225 @@
+
+Create a file named `dns.json` with the following content:
+
+```json title="dns.json"
+{
+  "Comment": "CREATE CNAME records",
+  "Changes": [
+    {
+      "Action": "CREATE",
+      "ResourceRecordSet": {
+        "Name": "bootstrapper-controller.<your domain name>",
+        "Type": "CNAME",
+        "TTL": 300,
+        "ResourceRecords": [
+          {
+            "Value": "ab4feaf807c9440b8b021395f0422b26-1385734990.us-east-2.elb.amazonaws.com"
+          }
+        ]
+      }
+    },
+    {
+      "Action": "CREATE",
+      "ResourceRecordSet": {
+        "Name": "api.<your domain name>",
+        "Type": "CNAME",
+        "TTL": 300,
+        "ResourceRecords": [
+          {
+            "Value": "a6ae77105db624f06bb37f18843db925-911888461.us-east-2.elb.amazonaws.com"
+          }
+        ]
+      }
+    },
+    {
+      "Action": "CREATE",
+      "ResourceRecordSet": {
+        "Name": "controller.<your domain name>",
+        "Type": "CNAME",
+        "TTL": 300,
+        "ResourceRecords": [
+          {
+            "Value": "ac63855b699064f5b825e707cfe290f0-1144570992.us-east-2.elb.amazonaws.com"
+          }
+        ]
+      }
+    },
+    {
+      "Action": "CREATE",
+      "ResourceRecordSet": {
+        "Name": "*.nms.<your domain name>",
+        "Type": "CNAME",
+        "TTL": 300,
+        "ResourceRecords": [
+          {
+            "Value": "a7b65619cbf8443feb05910823b6c42b-748407990.us-east-2.elb.amazonaws.com"
+          }
+        ]
+      }
+    }
+  ]
+}
+```
+
+!!! note
+    Replace each resource record value with the ones received from the previous step using the following Service/Hostname scheme:
+
+    | Service                                | Hostname                              | 
+    |----------------------------------------|---------------------------------------|
+    | `<orc8r-bootstrap-nginx External IP>`  | `bootstrapper-controller.<your domain name>` | 
+    | `<orc8r-nginx-proxy External IP>`      | `api.<your domain name>`                     | 
+    | `<orc8r-clientcert-nginx External IP>` | `controller.<your domain name>`              | 
+    | `<nginx-proxy External IP>`            | `*.nms.<your domain name>`                   | 
+
+
+Create the CNAME records in Route53:
+
+```console
+aws route53 change-resource-record-sets --hosted-zone-id <your hosted zone ID> --change-batch file://dns.json
+```
+
+Now, navigate to `https://master.nms.<your domain name>`, you should receive a warning because we are using self-signed-certificates, click on "Proceed".
+
+## Verifying the deployment
+
+Get the master organization's username and password:
+
+```console
+juju run-action nms-magmalte/leader get-master-admin-credentials --wait
+```
+
+Note the `admin-username` and `admin-password` values.
+
+Confirm successful deployment by visiting `https://master.nms.<your domain name>` and logging in with the `admin-username` and `admin-password` outputted here.
```

Comment:
I think you are missing the step for the `admin_operator.pfx` cert.

Reply:
The cert is not needed to connect to the UI and complete the tutorial, it would have been needed to make API calls.

Reply:
You mean that I have been changing the certificate in my browser each time I redeploy for nothing all this time?

Reply:
I was doing this as well. I'm wondering if we want to have this [section in the Orchestrator deployment](https://canonical.github.io/charmed-magma/1.6/how-to/deploy_charmed_magma_orchestrator/#import-the-admin-operator-https-certificate) if it's not required. I wouldn't remove it from the docs, but I'd probably create another section.

Reply:
That's a good point, I just moved it to its own section in the `how-tos`